### ISSUE TITLE
Fix: Disable iOS 18 TabBar

### ISSUE THREAD

**Issue by anduong96** [wix/react-native-navigation#7926]
### Motivation

Ios 18 introduced a new tab bar UI that overwrites the bottom tab UI
![](https://i.sstatic.net/fQOy0b6t.png)

This PR disable that

### Are you willing to resolve this issue by submitting a Pull Request?

- [ ] Yes, I have the time, and I know how to start.
- [X] Yes, I have the time, but I don't know how to start. I would need guidance.
- [ ] No, I don’t have the time and I’m okay to wait for the community / maintainers to resolve this issue.

**Comment by manuhook** [comment]
Thank you very much

**Comment by bumpingChris** [comment]
Hi @anduong96, Thanks for doing this. Any progress so far?

**Comment by aschmitz-wbd** [comment]
I wouldn't want this to be disabled outright. This is the native way iPad deals with tabs. I think there should be an option or it should stay as it is.

Btw. the iPad also dynamically updates to the bottom bar when the app is scaled down, e.g. when in a split window.

**Comment by Kioshia** [comment]
for me the biggest problem with that damn update is that IOS18 sets the TabBar on Top competly messing up my NavigationItems on top... what you just wrote that with the compact style kinda doesnt fix it for me cuase i have 8 NavigationTabs when i have more then 4 it just shows "More" which completly messes up my styling in the app i cant change that cause the app is super duper old so is there a way to just turn it back the os17 one ? I want my 8 Tabs displayed on the bottom...

**Comment by Meet-iottive** [comment]
Using your solution tab bar goes to bottom but tab bar items title and icons are disappearing any solution for that ? @anduong96

**Comment by Aikenwu** [comment]
Maybe use a stupid way to solve the problem, I get the bottom tab height, then if iPad and iOS 18, set all root page padding bottom, to avoid a hidden problem.

**Comment by anduong96** [comment]
> Using your solution tab bar goes to bottom but tab bar items title and icons are disappearing any solution for that ? [@anduong96](https://github.com/anduong96)

No, sorry. This code works for my use case. Im not looking to be a regular contributor.

**Comment by anduong96** [comment]
> I wouldn't want this to be disabled outright. This is the native way iPad deals with tabs. I think there should be an option or it should stay as it is.
> 
> Btw. the iPad also dynamically updates to the bottom bar when the app is scaled down, e.g. when in a split window.

Feel free to lift this code and make it optional
